pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -174,7 +175,37 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
   /** The current config state. */
   protected cfg: BackendSettings | undefined;
 
-  /** Indicates wherther the current installation is a privileged install or not. */
+  /** Indicates wherther the current installtion is an Admin Install. */
+  #isAdminInstall: Promise<boolean> | undefined;
+
+  protected getIsAdminInstall(): Promise<boolean> {
+    this.#isAdminInstall ??= new Promise((resolve) => {
+      let key;
+
+      try {
+        key = reg.openKey(reg.HKLM, 'SOFTWARE', reg.Access.READ);
+
+        if (key) {
+          const parsedValue = reg.getValue(key, 'SUSE\\RancherDesktop', 'AdminInstall');
+          const isAdmin = parsedValue !== null;
+
+          return resolve(isAdmin);
+        } else {
+          console.debug('Failed to open registry key: HKEY_LOCAL_MACHINE\SOFTWARE\SUSE\RancherDesktop');
```

Comment:
FWIW, on re-reading I see that we actually reach here if we fail to open `HKLM\SOFTWARE` (i.e. not even one of ours); so the message is a bit wrong. But whatever.